modules/ROOT/pages/_partials/commons/common-p2psync-websocket-using-passive.adoc

```diff
@@ -189,8 +189,8 @@ Use {url-api-class-urlendpointconfiguration}'s {url-api-references-urlendpointco
 
 Define whether the connection is to use TLS or clear text.
 
-TLS based encryption is enabled by default and this setting ought to be used in any production environment.
-However, it _can_ be disabled, for example for development and-or test environments.
+TLS-based encryption is enabled by default, and this setting ought to be used in any production environment.
+However, it _can_ be disabled, for example, for development and-or test environments.
```

Comment:
Again I think that comma is extraneous?
Could split new sentence before "for example" instead?

Also note "and-or" (a weird idiosyncracy you see sometimes in  mobile docs, should probably be "and/or" or just "or")